is-ios-safari.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+var isIos = require('./is-ios');
+var webkitRegexp = /WebKit/i;
```

Comment:
Because this is case insensitive, would `/webkit/i` be preferable? Minor point indeed.